ncm-metaconfig/src/main/metaconfig/slurm/pan/schema.pan

```diff
@@ -86,7 +86,7 @@ type slurm_scheduler_parameters = {
     'bf_yield_interval' ? long(0..)
     'bf_yield_sleep' ? long(0..)
     'build_queue_timeout' ? long(0..)
-    'default_queue_depth' ? long(0..)
+    '{default_queue_depth}' ? long(0..)
```

Comment:
Why are these keys being escaped in the schema?

Reply:
Have a look at the discussion in #1422

Reply:
I'm actually not 100% sure what the issue exactly is. @itkovian ?

Reply:
Ah yes, I don't understand it either!

Reply:
i think  all keys are unescaped, but this means this generates invalid code unless escaped

Reply:
When I drop the `{ }`, it fails with:

```
# -SchedulerParameters=bf_continue,bf_max_job_test=1024,bf_window=4320,default_queue�pth=128
# +SchedulerParameters=bf_continue,bf_max_job_test=1024,bf_window=4320,default_queue_depth=128
```

It smells like a bug? When doesn't this happen with other keys? For example `preempt_reorder_count` produces the correct result without `{ }`.

Reply:
this is not a bug, this the crappy unneeded escaping garbage in quattor. the escape function is so dumb, you get crap like this. anything that is `_[a-f][a-f]` can be unescaped, so you have to be really careful with simply escaping all because of the rare cornercases like ,_debug`, `_depth`, `_action` and a few others we encountered over the years.

Reply:
OK but when I flip
```
"convert/unescapekey" = true;
```
to `false` and drop the `{ }` it gives the correct result. But that would revert #1422 which I guess was opened for good reason?

Reply:
there are some keys here and there that do require escaping since they contain a `/`. 
imho, this weirdness is a small price to pay. the alternative is to write much more complicated TT files.

and all this for some bad design decision of over 10years old, really not worth it.

Reply:
OK, then we keep it as-is.

Other remarks @jrha ?

Reply:
Ok I finally get it, we can probably live with it.